src/components/table/TableView.svelte

```diff
@@ -27,11 +30,23 @@
   export let pageSize = 10;
   export let bucketTypeLabel = 'Categories';
 
+  let showHistogramData = false;
+
   let totalPages = 0;
   let currentPage = 0;
   $: if (data) currentPage = 0;
   $: totalPages = Math.ceil(data.length / pageSize);
 
+  const metricsWithPercentileData = [
+    'histogram-exponential',
+    'histogram-linear',
+    'keyed-scalar',
+    'scalar',
+    'quantity',
+    'counter',
+    'labeled_counter',
+  ];
+
```

Comment:
I prefer to uppercase constant definitions to make it more clear what they are. Also this is probably better expressed as a [set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set).

(at some point we might want to shove this type of thing off into a constants file, but maybe doesn't need to happen here)

```suggestion
  const METRICS_WITH_PERCENTILE_DATA = new Set([
    'histogram-exponential',
    'histogram-linear',
    'keyed-scalar',
    'scalar',
    'quantity',
    'counter',
    'labeled_counter',
  ]);

```